device/mellanox/x86_64-mlnx_msn2700-r0/Mellanox-SN2700-D44C10/buffers_defaults_t0.j2

```diff
@@ -0,0 +1,133 @@
+{#
+    Copyright (c) 2022 NVIDIA CORPORATION & AFFILIATES.
+    Apache-2.0
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+#}
+{% set default_cable = '5m' %}
+{% set ingress_lossless_pool_size =  '6822912' %}
+{% set ingress_lossless_pool_xoff =  '999424' %}
+{% set egress_lossless_pool_size =  '13945824' %}
+{% set egress_lossy_pool_size =  '6822912' %}
+
+{%- macro generate_port_lists(PORT_ALL) %}
```

Comment:
not sure whether we need this. 
for MSFT SKUs, the port configuration is generated based on the mini graph and the PORT_ALL can be rendered based on the port configuration. so we usually leave this macro undefined.

Reply:
Hi @stephenxs  Yes I went through the code and figured this out. However all SKUs do have it and wrongly defined. So I just made it aligned with the SKU. I don't think having it would cause any harm.

Reply:
You are correct. I thought I had removed the macro definition and left it empty but that is 202012 and above.
In 201911, we still have it. So let's keep it in 201911.
Approved.